### DISCUSSION TITLE
How can I remove picture from the cropper ?

### DISCUSSION THREAD

**Discussion by clemcoder** [advanced-cropper/vue-advanced-cropper#254] [Q&A]
Hello,
I try to completely reset the cropper on a button click.
How can I remove the selected picture from the cropper ?
Thank you for your help.

**Comment by Norserium** [comment]
@clemcoder, did you try to clear the `src` prop value?

**Comment by clemcoder** [comment]
Yes I did like this : 

currentCropper.src = null;

But here is the error :

vue.esm.js:5073 [Vue warn]: Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "src"

**Reply by Norserium** [reply]
You shouldn't do it directly. You pass some variable to the cropper component:
```jsx
<Cropper src={src} />
```

Clear that variable somewhere and that's all.
```js
this.src = null
```

**Comment by clemcoder** [comment]
Thank you for your help, it works.
A juste had to do this.$forceUpdate(); (maybe because my image is in an array).
